org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -36,47 +36,33 @@ $button-border-radius: 2px;
   .crm-form-block {
     &.crm-search-form-block > .crm-submit-buttons,
     &.crm-form-block > .crm-submit-buttons {
-      padding-top: $toolbar-top-space;
-      background: $crm-accordion-header-color !important;
-
-      %btn {
```

Comment:
I think @brunoqueiros has been working on this partials as well for his work on PCHR-1429, you might want to check his branch (and talk with him) to see if you're gonna have any conflicts down the line and just to make sure that the work you are both doing on this is consistent